yt/yt/server/cell_balancer/bundle_scheduler.cpp

```diff
@@ -1905,11 +1921,11 @@ THashMap<TSchedulerInputState::TQualifiedDCName, TDataCenterDisruptedState> GetD
     return result;
 }
 
-TInstanceAnnotationsPtr GetInstanceAnnotationsToSet(
+TBundleControllerAnnotationsPtr GetBundleControllerAnnotationsToSet(
```

Comment:
I would like to retain the word `Instance` in this method, how about `GetInstanceBundleControllerAnnotationsToSet`? Rather clumsy, yet it is of limited use so should not be problematic.

Reply:
In order to match the type name, the method name `GetBundleControllerInstanceAnnotationsToSet` is better.